x-pack/plugins/uptime/public/components/overview/monitor_list/columns/monitor_tags.tsx

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
Let's extract this component to a common directory for shared stuff. I'd expect it to only be used in the monitor list given its location in `components/overview/monitor_list` but it's also used in the status bar for the status details.

Reply:
done